_events/2023-0718-2-9-release-meetings.markdown

```diff
@@ -2,7 +2,6 @@
 
 
 eventdate: 2023-07-19 15:00:00 -0800
```

Comment:
Also, the page says "July 19th - 3PM PT" bellow, and PT is currently PDT, that is -0700 and not -0800…

```suggestion
eventdate: 2023-07-19 15:00:00 -0700
```

TZ abbreviations are evil :smiling_imp: !